core/util/flask_util.py

```diff
@@ -173,3 +173,9 @@ class OPDSEntryResponse(Response):
     def __init__(self, response=None, **kwargs):
         kwargs.setdefault("mimetype", OPDSFeed.ENTRY_TYPE)
         super().__init__(response, **kwargs)
+
+
+def boolean_value(value):
+    """Convert a string request value into a boolean, used for form encoded requests
+    JSON encoded requests will get automatically converted"""
+    return True if value in ["true", "True", True] else False
```

Comment:
It would be good to have some additional tests for "truthy" values for this condition: especially non-empty strings like "false", but also non-zero number, etc.